cinema/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField()
+    description = serializers.CharField(style={"type": "textarea"})
+    duration = serializers.IntegerField()
+
+    def create(self, validated_data):
+        return Movie.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.title = validated_data.get("title", instance.title)
+        instance.description = validated_data.get(
+            "description", instance.description
+        )
+        instance.duration = validated_data.get("duration", instance.duration)
+        return instance
```

Comment:
use ModelSerializer, it includes both create() and update() methods from box

Reply:
Done, but CBV is topic of the next lesson